tool/tctl/common/helpers_test.go

```diff
@@ -212,6 +219,8 @@ func makeAndRunTestAuthServer(t *testing.T, opts ...testServerOptionFunc) (auth
 	cfg.Proxy.DisableWebInterface = true
 	cfg.InstanceMetadataClient = cloud.NewDisabledIMDSClient()
 	auth, err = service.NewTeleport(cfg)
+	c := clockwork.NewFakeClock()
+	auth.GetAuthServer().SetClock(c)
```

Comment:
Are you sure this doesn't break any other tests?

Reply:
I've made this conditional on a new flag `useFakeClock`